extensions/dacpac/src/wizard/pages/deployConfigPage.ts

```diff
@@ -195,4 +177,70 @@ export class DeployConfigPage extends DacFxConfigPage {
 		this.databaseLoader.loading = false;
 		return true;
 	}
+
+	/*
+	Function that is used to change the radio button DOM based on whether databases exist or not.
+	*/
+	private async disableUpgradeRadioButton(): Promise<azdata.FormComponent> {
+		/* Set the upgrade radio button to be disabled and call the updateNewRadioButton function
+		to update the new radio button accordingly.
+		*/
+		this.upgradeRadioButton.enabled = false;
+		this.newRadioButton.checked = true;
+
+		this.updateNewRadioButton();
+
+		return this.displayRadioButtons(this.upgradeRadioButton, this.newRadioButton);
+	}
+
+	/*
+	Function that is used to update the window if upgrade radio button is selected.
+	*/
+	private async updateUpgradeRadioButton(): Promise<void> {
```

Comment:
nothings being awaited here, so I don't think it needs to be an async function. Same with updateNewRadioButton() below

Reply:
Acknowledged, thank you!